examples/csv-streams-to-record-arrays/csv_streams_to_record_array.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/data.csv;
+import ballerina/io;
+
+type Book record {
+    string name;
+    string author;
+    decimal price;
+    string publishedDate;
+};
+
+type BookMinimal record {|
+    string name;
+    string author;
+|};
+
+const csvFilePath = "./files/csv_file.csv";
+
+public function main() returns io:Error? {
+    // Write a CSV file.
+    check io:fileWriteCsv(csvFilePath, [["name", "author", "price", "publishedDate"],
+                                        ["Effective Java", "Joshua Bloch", "45.99", "2018-01-01"],
+                                        ["Clean Code", "Robert C. Martin", "37.5", "2008-08-01"]]);
```

Comment:
Why not just add the file separately?

Reply:
Can we do that in the bbes, In the `io-bbes` also, did the same approach
https://ballerina.io/learn/by-example/io-csv/